web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -87,14 +88,25 @@
             )
         );
 
-        resourceLinks.add(
-            ResourceLinkDTO.create(
-                ResourceLinkTypeDTO.SIGN_CERTIFICATE,
-                "Signera",
-                "Signerar intygsutkast",
-                true
-            )
-        );
+        if (certificate.getMetadata().getType().equalsIgnoreCase(Af00213EntryPoint.MODULE_ID)) {
+            resourceLinks.add(
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.SIGN_CERTIFICATE,
+                    "Signera och skicka",
+                    "Signerar intygsutkast",
```

Comment:
The description should be adjusted also to reflect send and sign. Something like:
            'common.signsend.tooltip': 'Intyget skickas direkt till ${recipient}.',
            
But we could generalize it to be "intygsmottagare" instead of specific recipient